frontend/js/src/settings/Settings.tsx

```diff
@@ -73,7 +98,7 @@ export default function Settings() {
         currentUser?.auth_token,
         selectedFlair
       );
-      toast.success("Flair preferences updated successfully");
+      toast.success("Flair saved successfully");
```

Comment:
Now, on updaing the flair preference, the cache needs to invalidated so that it gets updated everywhere.

```suggestion
      queryClient.invalidateQueries({ queryKey: ["flair"] });
```